modules/account-lib/src/coin/xrp/iface.ts

```diff
@@ -0,0 +1,33 @@
+export interface SignedXRPTransaction {
+  signedTransaction: string;
+  id: string;
+}
+
+export interface ApiMemo {
+  MemoData?: string;
```

Comment:
Shouldn't these start with lower case?

Reply:
ApiMemos are based on the `TypeDef` from the Ripple SDK. Link to [Memos](https://xrpl.org/transaction-common-fields.html#memos-field).